README.md

```diff
@@ -1,2 +1,29 @@
-# Portfolio-setup-and-mobile-version-skeleton
-My Protfolio
+# Project Name
+
+Portfolio-setup-and-mobile-version-skeleton
+
+# PR
+
+basic design of first section in my portfolio
+
+![screenshot](./Capture.png)
+
+The very first siction of the mobile verstion of my portfolio
+
+## Built With
+
+- Html
+- css
+- Litner
+
```

Comment:
- [ ] Please add another title (like **# Getting Started** or **# Setup**) and accompanying description on how users can get a local copy of your project. Something like this 👇 👇 should be okay:
```
## Getting Started

### Setup

#### To get a local copy up and running follow these simple example steps.

- Click the green `Code` button on the repo and copy link
- In your local PC, open your terminal in the folder you would like to clone the project.
- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton.git`
- On the terminal, navigate into the directory like so: `cd Portfolio-setup-and-mobile-version-skeleton`
```